modules/windows-workload-management.adoc

```diff
@@ -5,7 +5,7 @@
 [id="windows-workload-management_{context}"]
 = Windows workload management
 
-To run Windows workloads in your cluster, you must first install the Windows Machine Config Operator (WMCO). The WMCO is a Linux-based Operator that runs on Linux-based control plane and compute nodes. The WMCO orchestrates the process of deploying and managing Windows workloads on a cluster.
+To run Windows workloads in your cluster, you must first install xref:../windows_containers/windows-containers-release-notes-5-x.adoc#windows-containers-release-notes-5-x (WMCO). The WMCO is a Linux-based Operator that runs on Linux-based control plane and compute nodes. The WMCO orchestrates the process of deploying and managing Windows workloads on a cluster.
```

Comment:
Modules aren't allowed to have xrefs

Reply:
What happened here? Thanks for catching this.